doc/sphinx/install.rst

```diff
@@ -98,14 +99,18 @@ Install and configure the search engine
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 dissem.in uses the `Elasticsearch <https://www.elastic.co/products/elasticsearch>`_
-backend for Haystack.
+backend for Haystack. The current supported version is **5.x.x**.
 
 `Download Elasticsearch <https://www.elastic.co/downloads/elasticsearch>`_
 and unzip it::
 
     cd elasticsearch-<version>
     ./bin/elasticsearch    # Add -d to start elasticsearch in the background
 
+Alternatively you can install the .rpm or .deb package, see the documentation of Elasticsearch for further information.
+
+Make sure to set the initial `heapsize https://www.elastic.co/guide/en/elasticsearch/reference/5.6/heap-size.html` accordingly.
```

Comment:
I think there are some `<` / `>` missing here.

Reply:
Done in https://github.com/dissemin/dissemin/pull/566/commits/ec9032654f40dcd5d129c811a2e9a5b9fb31fd48